kubernetes/src/main/java/io/kubernetes/client/custom/Pair.java

```diff
@@ -0,0 +1,20 @@
+package io.kubernetes.client.custom;
+
+public class Pair<L, R> {
```

Comment:
We already depend on apache-commons, rev the version to 3.0 and use the Pair implementation from there?